tsgen/__main__.py

```diff
@@ -152,7 +152,7 @@ def generate_on_car_telemetry(self):
                  * 
                  * @param[in]   index   Index in table
                  */
-                const can_handler_t* can_handler_get(size_t index)
+                const can_handler_t* can_handler_get(uint32_t index)
```

Comment:
I'm undecided whether `size_t` or `uint32_t` is better practice on an embedded platform, but let's just go with `uint32_t`. Doesn't really matter!